ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -1532,7 +1530,7 @@ class EclCmdAbort : public EclCmdCommon
             reqQ->setJobname(optName.get());
 
             Owned<IClientWUQueryResponse> respQ = client->WUQuery(reqQ);
-            int res = respQ->queryClientStatus();
+            [[maybe_unused]] int res = respQ->queryClientStatus();
```

Comment:
I don't know what the purpose of this code is well enough to know whether the result SHOULD be bing used, but it feels like it should - so this may indicate a bug.